jekyll/_cci2/server-3-install-prerequisites.adoc

```diff
@@ -45,99 +68,310 @@ CircleCI server users: +
 
-| https://kots.io/[Kots]
-| {kotsversion} or greater
-| Replicated Kubernetes Application Management
-|===
+CAUTION: Do not use Autopilot cluster. CircleCI requires functionality that is not supported by GKE Autopilot.
 
+NOTE: In the following steps replace any items credentials displayed between `< >` with your details.
 
-## External Ports
-[.table.table-striped]
-[cols=6*, options="header", stripes=even]
-|===
-| Port number
-| Protocol
-| Direction
-| Source / Destination
-| Use
-| Notes
-
-| 80
-| TCP
-| Inbound
-| End users
-| HTTP web app traffic
-|
-
-| 443
-| TCP
-| Inbound
-| End users
-| HTTP web app traffic
-|
-
-| 8800
-| TCP
-| Inbound
-| Administrators
-| Admin console
-|
-
-| 22
-| TCP
-| Inbound
-| Administrators
-| SSH
-| Only required for the bastion host
-
-| 64535-65535
-| TCP
-| Inbound
-|
-| SSH into builds
-| Only required for the nomad clients.
-|===
+. https://cloud.google.com/sdk/gcloud[Install] and https://cloud.google.com/kubernetes-engine/docs/quickstart#defaults[configure] the GCP CLI for your GCP account. This includes creating a Google Project, which will be required to create a cluster within your project. When you create your project make sure you also enabled API access. If you do not enable API access, the command we will run next, to create your cluster, will fail. 
+. Create your cluster by entering running the following: 
++
+```sh
+gcloud container clusters create circleci-server --project <YOUR_GOOGLE_CLOUD_PROJECT_ID> --region europe-west1 --num-nodes 3 --machine-type n1-standard-4
+``` 
+. Configure` kubectl` with your your credentials gcloud credentials: 
++
+```sh
+gcloud container clusters get-credentials circleci-server --region europe-west1
+```
+. Verify your cluster: 
++
+```sh
+kubectl cluster-info
+```
+. Create a service account for this cluster: 
++
+```sh
+gcloud iam service-accounts create <YOUR_SERVICE_ACCOUNT_ID> --description="<YOUR_SERVICE_ACCOUNT_DISPLAY_NAME>"  --display-name="<YOUR_SERVICE_ACCOUNT_DISPLAY_NAME>"
+```
+. Get the credentials for the service account: 
++
+```sh
+gcloud iam service-accounts keys create <PATH_TO_STORE_CREDENTIALS> --iam-account <SERVICE_ACCOUNT_ID>@<YOUR_GOOGLE_CLOUD_PROJECT_ID>.iam.gserviceaccount.com
+```
+
+=== Create a new GitHub OAuth app
+Registering and setting up a new GitHub OAuth app for CircleCI server allows for authorization control to your server installation using GitHub OAuth and for updates to GitHub projects/repos using build status information.
+
+. In your browser navigate to **your GitHub instance** > **Settings** > **Developer Settings** > **OAuth Apps** and click the **New OAuth App** button. 
+
+. Complete the following fields based on your planned installation: 
+** Homepage URL: The URL of your planned CircleCI installation.
+** Authorization callback URL: The authorization callback URL will be the URL of your planned CircleCI installation followed by /auth/github 
+
+=== Frontend TLS certificates
```

Comment:
Would be good to link to this section from the KOTS admin console where it says "see documentation"
<img width="756" alt="Screenshot 2021-09-23 at 11 39 46" src="https://user-images.githubusercontent.com/24589403/134493695-a1f08f80-9c07-40f6-80e1-951ca19ba13d.png">
:

Reply:
Also what happens if they uncheck this box - do we need further docs for this scenario?

Reply:
@nathanfishcircleci